pysteps/tracking/tdating.py

```diff
@@ -211,12 +211,12 @@ def tracking(cells_id, cells_id_prev, labels, V1, max_ID):
                 size.append(len(x))
             biggest = np.argmax(size)
             new_ID = new_ID[biggest]
-            cells_id_new.ID.iloc[ID] = new_ID
+            cells_id_new.loc[index, "ID"] = new_ID
```

Comment:
using `.loc[index, "ID"]` instead of `.ID.iloc[ID]` avoids the warning by removing the chaining operation. In this case, `.loc` should be more suited than `.iloc` since `index`refers to the index label and not the index position (although the two are in fact the same value...).